packages/dnb-eufemia/src/components/accordion/AccordionHeader.tsx

```diff
@@ -97,15 +98,27 @@ export type AccordionHeaderIconProps = {
   icon?: AccordionHeaderIconIcon
   size?: IconSize
   expanded?: boolean
+  iconRight?: boolean
```

Comment:
```suggestion
  iconPosition?: 'left' | 'right'
```

With default to `left`?